src/containers/Projects/ProjectRulesAndGuide/MilestonesAndPayouts/MilestonesAndPayouts.scss

```diff
@@ -0,0 +1,231 @@
+        margin-right: 12px;
+      }
+    }
+
+    &-text {
+      color: var(--color-gray-700);
+      font-size: 14px;
+      line-height: 143%;
+      margin: 0;
+    }
+  }
+
+  &__pow {
+    margin-bottom: 88px;
+
+    &-heading {
+      color: var(--color-gray-900);
+      font-size: 20px;
+      font-weight: 700;
+      line-height: 120%;
+      margin-bottom: 64px;
+    }
+
+    &-table {
+      &-row {
+        box-shadow: inset 0px -1px 1px #e6e6e6;
+        display: grid;
+        gap: 32px;
+        grid-template-columns: 160px auto 160px;
+        padding: 32px 40px;
+
+        &-apps {
+          align-self: center;
+          display: flex;
+          justify-self: end;
+
+          &-logo {
+            align-items: center;
+            background: var(--color-blue-50);
+            border-radius: 50%;
+            display: flex;
+            height: 56px;
+            justify-content: center;
+            width: 56px;
+          }
+        }
+
+        &-heading {
+          align-self: center;
+          border-right: 1px solid var(--color-gray-075);
+          color: var(--color-blue-900);
+          font-size: 16px;
+          line-height: 120%;
+          padding-right: 8px;
+        }
+
+        &-text {
+          align-self: center;
+          color: var(--color-gray-700);
+          font-size: 16px;
+          line-height: 120%;
+          margin: 0;
+        }
+      }
+    }
+  }
+
+  &__process {
+    margin-bottom: 120px;
+
+    &-heading {
+      color: var(--color-gray-900);
+      font-size: 20px;
+      font-weight: 700;
+      line-height: 120%;
+      margin-bottom: 64px;
+    }
+
+    &-step {
+      box-shadow: inset 0px -1px 1px #e6e6e6;
+      display: grid;
+      gap: 32px;
+      grid-template-columns: 56px auto;
+      padding: 32px;
+
+      &-text {
+        align-self: center;
+        color: var(--color-gray-700);
+        font-size: 16px;
+        line-height: 120%;
+      }
+    }
+  }
+
+  &__roadmap {
+    box-shadow: 0px 0px 30px rgba(229, 234, 244, 0.48);
+    margin-bottom: 120px;
+    padding: 40px;
+
+    &-heading {
+      color: var(--color-gray-900);
+      font-size: 20px;
+      font-weight: 700;
+      line-height: 120%;
+      margin-bottom: 8px;
+    }
+
+    &-illustration {
+      height: auto;
+      width: 100%;
+    }
+
+    &-text {
+      color: var(--color-gray-700);
+      font-size: 16px;
+      line-height: 120%;
+      margin-bottom: 88px;
+    }
+  }
+
+  @media (max-width: 992px) {
+    &__roadmap {
+      padding: 48px;
+    }
+  }
+
+  @media (max-width: 768px) {
+    &__pow {
+      &-table {
+        &-row {
+          padding: 28px 32px;
+        }
+      }
+    }
+
+    &__process {
+      &-divider {
+        border-top: 1px solid #e6e6e6;
+        margin: 0;
+        width: 60%;
+      }
+
+      &-step {
+        box-shadow: none;
+        padding: 32px 0;
+      }
+    }
+
+    &__roadmap {
+      padding: 32px;
+    }
+  }
+
+  @media (max-width: 414px) {
```

Comment:
btw @thesanjeevsharma me and @angle943 clarified with MR K recently about the meanign of hte breakpoints, it should be `min-width` instead. We've made a mistake for the past features thinking that it was `max-width` 😅 [discord thread](https://discord.com/channels/774574985510518806/823383842844180561/854524278573367296)

Reply:
Let me try updating these. :(